index.bs

```diff
@@ -678,6 +678,9 @@ This set is initially [=set/is empty|empty=].
 A [=sensor=] has an associated <dfn>latest reading</dfn> [=ordered map|map=]
 which holds the latest available [=sensor readings=].
 
+Any time the [=latest reading=] is updated, the UA invokes [=Report latest reading updated=]
```

Comment:
invokes [=Report latest reading updated=] (operation | algorithm) for every... wdyt?